### PR TITLE
Fix CVEs CVE-2022-21698 CVE-2022-27191

### PR DESCRIPTION
**Pull request by tuxtof**
**What this PR does / why we need it**:

fix following CVEs:
- CVE-2022-27191
- CVE-2022-21698

**Release note**:
```release-note
fix CVE-2022-21698
fix CVE-2022-27191
```

**Test**

```
> make test-e2e
Ran 10 of 20 Specs in 1643.450 seconds
SUCCESS! -- 10 Passed | 0 Failed | 0 Pending | 10 Skipped
PASS
```

### CODE REVIEW

**Review by vnephologist — APPROVED** [review]
LGTM

File: go.mod [hunk 1]
```diff
@@ -135,3 +135,7 @@ require (
 	sigs.k8s.io/structured-merge-diff/v4 v4.2.1 // indirect
 	sigs.k8s.io/yaml v1.3.0 // indirect
 )
+
+replace golang.org/x/crypto => golang.org/x/crypto v0.0.0-20220314234659-1baeb1ce4c0b
+
+replace github.com/prometheus/client_golang => github.com/prometheus/client_golang v1.11.1
```
Comment (thunderboltsid) [inline go.mod:141]: consider making it a block:

```go
replace (
    golang.org/x/crypto => golang.org/x/crypto v0.0.0-20220314234659-1baeb1ce4c0b
    github.com/prometheus/client_golang => github.com/prometheus/client_golang v1.11.1
)
```

File: go.mod [hunk 2]
```diff
@@ -135,3 +135,7 @@ require (
 	sigs.k8s.io/structured-merge-diff/v4 v4.2.1 // indirect
 	sigs.k8s.io/yaml v1.3.0 // indirect
 )
+
+replace golang.org/x/crypto => golang.org/x/crypto v0.0.0-20220314234659-1baeb1ce4c0b
```
Comment (thunderboltsid) [inline go.mod:139]: The problem with such statements, in general, is that they end up sticking the version to a specific version which inevitably causes more exposure to CVEs. We need to be mindful that we are pinning the versions to a specific one and remove them once no longer needed.

**Review by thunderboltsid — DISMISSED** [review]

File: go.mod (hunk 2 above)
Comment (tuxtof) [reply go.mod]: fully agree , but for the moment this is the only solution to respect what security's team is asking us

File: go.mod (hunk 1 above)
Comment (tuxtof) [reply go.mod]: fixed

**Review by thunderboltsid — APPROVED** [review]

**Review by yannickstruyf3 — APPROVED** [review]

**Pull request merged**